fitbenchmarking/controllers/scipy_go_controller.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, cost_func):
         self._popt = None
         self._status = None
         self._maxiter = None
+        self._kwargs = {}
```

Comment:
It does not need to be at the class level necessarily. Pylint complains because `self._kwargs` is being created in if else clauses and used later. It just needs to be declared somewhere as an empty dict. I did it in the init because other variables were being declared there. I think it is also how it is being done in other controllers. But if I declare it as a function variable, it is a more optimized because access to it will be faster. I can update this to be in the scope of the function.

<img width="549" alt="image" src="https://github.com/user-attachments/assets/1215b3c2-7255-4df9-8cc1-96a8fd66422a">